espanol/jordiHurtado/muertedejordi/muertedejordi.md

```diff
@@ -0,0 +1,5 @@
+Mientras asestaba el golpe mortal tuve una revelación "estaba a punto de asestar un golpe mortal a una pieza angular del universo!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.